templates/details/overview.html

```diff
@@ -73,7 +73,14 @@ <h4 class="p-heading--5 u-no-margin--bottom">Discuss this {{ package.type }}</h4
       <p>Share your thoughts on this charm with the community on discourse.</p>
       <p><a class="p-button--neutral" href="https://discourse.charmhub.io/">Join the discussion</a></p>
     </div>
-    <div class="col-9 col-large-6">
+    <div class="col-9">
+      {% if show_notification %}
+        <div class="p-notification--caution">
+          <p class="p-notification__response" style=" background-image: none; padding-left: 1rem;">
+            While many Reactive Framework charms work on machines today, it&rsquo;s recommended to create new charms with the Operator Framework. <a href="https://discourse.charmhub.io/t/a-brief-history-of-juju-and-charm-development/4474">Learn more about the history of charms</a>.
```

Comment:
Why not add a link to get started developing an Operator Framework charm?

Reply:
@anthonydillon I believe these were the links that @danielmutis was told to include